setup.py

```diff
@@ -30,7 +30,9 @@
         'Topic :: Utilities',
     ],
     platforms='any',
-    packages=find_packages(),
+    packages=find_namespace_packages(where='src/', include=['sphinxcontrib.inmanta']),
```

Comment:
Looking at the [example namespace package](https://github.com/pypa/sample-namespace-packages/blob/master/native/pkg_b/setup.py), I *think* this could be simply
```suggestion
    packages=['sphinxcontrib.inmanta'],
```